bin/public-installer-releaser.sh

```diff
@@ -5,15 +5,9 @@ SET_AS_LATEST=${SET_AS_LATEST:-true}
 
 COMMIT_HASH=${COMMIT_HASH:-"$(git rev-parse --short HEAD || true)"}
 
-# This will pin the latest stable version.manifest so that the script will always run with
-# the same version of version.manifest.
-echo "Fetching latest stable to src/build/version.manifest..."
```

Comment:
we were incorrectly fetching the latest stable again, instead of using the existing fetched.